packages/css/src/components/document-list/document-list.scss

```diff
@@ -0,0 +1,63 @@
+.dso-document-list {
+  @include list-unstyled();
+
+  margin: 0 $grid-gutter-width * -0.5 0 $grid-gutter-width * -0.5;
+}
+
+.dso-document-list-item {
+  background-color: $grijs-5;
+  padding: $grid-gutter-width * 0.5;
+
+  a {
+    color: $zwart;
+  }
+
+  > :first-child {
+    margin: 0;
+  }
+
+  .dso-document-list-item-container {
+    position: relative;
+  }
+
+  .dso-document-list-item-type,
+  .dso-document-list-item-owner {
+    margin: 0 0 $u1 0;
+    display: inline-block;
+  }
+
+  .dso-document-list-item-status {
+    dso-badge,
+    .dso-badge {
+      margin-right: $u1;
+    }
+  }
+}
+
+dso-responsive-element[small] > .dso-document-list {
```

Comment:
`dso-responsive-element` is een element waarmee bestaande componenten worden verrijkt:

```html
<dso-responsive-element class="dso-document-header">
 [..]
</dso-responsive-element>
```

Maar dat kan niet overal: `<ul class="dso-document-list">`. Dat heb ik zo opgelost:

```html
<dso-responsive-element>
  <ul class="dso-document-list">
    [..]
   </ul>
</dso-responsive-element>
```

en met de selector op deze file. Het is belangrijk componenten een absolute selector naar `dso-responsive-element` hebben:

```scss
dso-responsive-element[small] > .dso-my-component {}
dso-responsive-element[small].dso-my-component {}
```

Dit is belangrijk zodat `<dso-responsive-element>` genest kan worden gebruikt zonder dat styling van het ene component actief wordt op het andere component.